### DISCUSSION TITLE
about compiler

### DISCUSSION THREAD

**Discussion by rraammiinn** [vlang/v#17429] [Questions and Answers]
what's pros and cons of using c or llvm for compile a language and why vlang uses c ?
dose it causes any limitation ?

**Comment by JalonSolov** [comment]
https://github.com/vlang/v/wiki/On-the-benefits-of-using-C-as-a-language-backend

**Comment by rraammiinn** [comment]
can I use avr headerfiles like mega32.h in a vlang program and compile it for atmega32 microcontroller ?
